mindocr/postprocess/det_fce_postprocess.py

```diff
@@ -302,12 +304,13 @@ def __init__(
     def _postprocess(self, pred, **kwargs):
         score_maps = []
         for value in pred:
-            value = value.asnumpy()
+            if not isinstance(value, Tensor):
```

Comment:
```suggestion
            if isinstance(value, Tensor):
```
Shouldn't it be this way?